applications/orochi-network-orosign-part1.md

```diff
@@ -0,0 +1,169 @@
+- **Contact Email:** hubert@orochi.network
+- **Website:** www.orochi.network
+
+### Legal Structure
+
+- **Registered Address:** OROCHI NETWORK PTE. LTD,68 CIRCULAR ROAD , #02-01 , SINGAPORE (049422)
+- **Registered Legal Entity:** OROCHI NETWORK PTE. LTD,68 CIRCULAR ROAD , #02-01 , SINGAPORE (049422)
+
+### Team's experience
+
+We are focusing in cryptography and especially ZKP, we want to utilize cryptography to provide **Verifiable Computation**.
+
+- Our technical blog [https://docs.orochi.network/](https://docs.orochi.network/)
+- Our Github repo [https://github.com/orochi-network](https://github.com/orochi-network)
+- Our proposal for zkMemory [https://hackmd.io/@chiro-hiro/SkqNGtcW2](https://hackmd.io/@chiro-hiro/SkqNGtcW2)
+
+### Team Code Repos
+
+- https://github.com/orochi-network
+
+### Team LinkedIn Profiles (if available)
+
+- [Chiro Hiro](https://www.linkedin.com/in/chiro8x)
+- [Hubert Nguyen](https://www.linkedin.com/in/hungnguyen88)
+
+## Development Status :open_book:
+
+[https://github.com/orochi-network](https://github.com/orochi-network)
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 5
+- **Full-time equivalent (FTE):** 5.2
+- **Total Costs:** $29,400
+
+### Milestone 1
+
+- **Estimated Duration:** 2 months
+- **FTE:** 1.6
+- **Costs:** $11,200
+
+| Number  | Deliverable | Specification                                                                                                                                              |
+| ------- | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **0a.** | License     | CC0 1.0                                                                                                                                                    |
+| **0b.** | Research    | Researching about Polkadot signature system and research their compatibility with MPC by which we can be fully compatible with Polkadot and its parachains |
+| **0c.** | Research    | Public article for every research we made, everything published under [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/)                        |
+| **1.**  | Research    | Researching about MPC in ECDSA (all supported signatures by Polkadot)                                                                                      |
+| **2.**  | Research    | Researching MPC for `secp256k1` and providing the document that describe how the MPC will be built and its security consideration                          |
+| **3.**  | Research    | Researching MPC for `ed25519` and providing the document that describe how the MPC will be built and its security consideration                            |
+| **4.**  | Research    | Researching MPC in `sr25519` and providing the document that describe how the MPC will be built and its security consideration                             |
```

Comment:
I'd still love to see the research deliverables expanded more here. 

- What questions are you looking to find answers to? 
- What problems currently exist that you are looking to solve?
- Have you already consulted with other MPC projects in the substrate ecosystem to identify common pain points among devs?
- Have you looked for other research in this area to be sure it still needs to be done?
- Can you explain what a semi-retail user is?

I understand that MPC is the next generation of seed phrase abstraction, but I'm trying to understand what this research is attempting to accomplish and what will be produced as a result of it. Thanks!